src/java/picard/analysis/MeanQualityByCycle.java

```diff
@@ -56,7 +56,7 @@
         usageShort = MeanQualityByCycle.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
-public class MeanQualityByCycle extends SinglePassSamProgram {
+public final class MeanQualityByCycle extends SinglePassSamProgram {
```

Comment:
This class doesn't have a private constructor - it just has a default no-arg public constructor.